res/lang/bossbetty/bossbetty_en.txt

```diff
@@ -1,153 +1,153 @@
 DIALOGUE_BETTY_PRESENT_WELCOME_LEVEL0_TEXT2   = My pleasure.
 DIALOGUE_BETTY_PRESENT_WELCOME_LEVEL1_TEXT1   = You're welcome, my dear.
 DIALOGUE_BETTY_PRESENT_WELCOME_LEVEL1_TEXT2   = I'll do anything for you.
 DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL0_TEXT1  = How dare you?
-DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL0_TEXT2  = You're disappointing me!
-DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL1_TEXT1  = You're disappointing me!
+DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL0_TEXT2  = What a disappointment!
+DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL1_TEXT1  = Sigh... what a disappointment.
 DIALOGUE_BETTY_PRESENT_HOW_DARE_LEVEL1_TEXT2  = Oh, how could you...
-DIALOGUE_BETTY_PRESENT_WELL1                  = Uhmm...
+DIALOGUE_BETTY_PRESENT_WELL1                  = Ummm...
 DIALOGUE_BETTY_PRESENT_WELL2                  = But...
 
-DIALOGUE_BETTY_ASK_FOR_SAMMYINFORMATION = How about the current Sammy?
+DIALOGUE_BETTY_ASK_FOR_SAMMYINFORMATION = What do you know about the current Sammy?
 DIALOGUE_BETTY_ASK_FOR_MASTERKEY        = You couldn't, by any chance, lend me the master key?
-DIALOGUE_BETTY_CHANGETOPIC              = Betty, let's talk about something else
-DIALOGUE_BETTY_LEVEL0_CHANGETOPIC1      = Let's talk about something else
-DIALOGUE_BETTY_GOODBYE                  = Hmm, see you.
-DIALOGUE_BETTY_LEVEL0_GOODBYE1          = Uhmm, goodbye
-DIALOGUE_BETTY_LEVEL1_GOODBYE1          = See you again
-DIALOGUE_BETTY_LEVEL1_GOODBYE2          = Hey, I have to go. See you later.
-DIALOGUE_BETTY_LEVEL1_GOODBYE3          = Listen Betty, work is calling. I have to go out for a minute.
+DIALOGUE_BETTY_CHANGETOPIC              = Betty, let's talk about something else.
+DIALOGUE_BETTY_LEVEL0_CHANGETOPIC1      = Let's talk about something else.
+DIALOGUE_BETTY_GOODBYE                  = Hmm... see you around.
+DIALOGUE_BETTY_LEVEL0_GOODBYE1          = OK... goodbye!
+DIALOGUE_BETTY_LEVEL1_GOODBYE1          = See you again!
+DIALOGUE_BETTY_LEVEL1_GOODBYE2          = Hey, I have to go. See you later!
+DIALOGUE_BETTY_LEVEL1_GOODBYE3          = Listen Betty, work's calling. I have to go out for a bit.
 
-BETTY_PRESENT_1            = foot spray
-BETTY_PRESENT_2            = dinner for two
-BETTY_PRESENT_3            = nose surgery
-BETTY_PRESENT_4            = custom written novel
-BETTY_PRESENT_5            = pearl necklace
-BETTY_PRESENT_6            = dwarf hamster coat
-BETTY_PRESENT_7            = necklace with large blood diamond
-BETTY_PRESENT_8            = sports car
-BETTY_PRESENT_9            = ring
-BETTY_PRESENT_10           = yacht
+BETTY_PRESENT_1            = Foot spray
+BETTY_PRESENT_2            = Dinner for two
+BETTY_PRESENT_3            = Nasal plastic surgery
+BETTY_PRESENT_4            = Personalized novel
+BETTY_PRESENT_5            = Pearl necklace
+BETTY_PRESENT_6            = Dwarf hamster coat
+BETTY_PRESENT_7            = Large red diamond necklace
+BETTY_PRESENT_8            = Sports car
+BETTY_PRESENT_9            = Diamond ring
```

Comment:
OK.  Ring just sounded a bit boring for a super expensive present.

Reply:
"Luxorious ring" or "Precious ring" or ... but I hope you got the point that I wanted to avoid a bit the textual "link" to the blood diamond present.

Reply:
OK no problem.